docs/configuration.md

```diff
@@ -654,6 +654,10 @@ responders:
 type: <tmpl_string>
 ```
 
+The `teams` responder is configured using the `name` field. This field can
```

Comment:
can you move this after L653 so it's more visible to users?

Reply:
@simonpasquier as a comment inside the "yaml" block then?

Reply:
yes perfect!